sdk/all/src/test/java/io/opentelemetry/sdk/OpenTelemetrySdkTest.java

```diff
@@ -306,6 +345,14 @@ void stringRepresentation() {
                 + "resource=Resource{schemaUrl=null, attributes={service.name=\"otel-test\"}}, "
                 + "spanLimitsSupplier=SpanLimitsValue{maxNumberOfAttributes=128, maxNumberOfEvents=128, maxNumberOfLinks=128, maxNumberOfAttributesPerEvent=128, maxNumberOfAttributesPerLink=128, maxAttributeValueLength=2147483647}, "
                 + "sampler=ParentBased{root:AlwaysOnSampler,remoteParentSampled:AlwaysOnSampler,remoteParentNotSampled:AlwaysOffSampler,localParentSampled:AlwaysOnSampler,localParentNotSampled:AlwaysOffSampler}, "
-                + "spanProcessor=SimpleSpanProcessor{spanExporter=MultiSpanExporter{spanExporters=[MockSpanExporter{}, MockSpanExporter{}]}}}}");
+                + "spanProcessor=SimpleSpanProcessor{spanExporter=MultiSpanExporter{spanExporters=[MockSpanExporter{}, MockSpanExporter{}]}}"
+                + "}, "
+                + "meterProvider=SdkMeterProvider{"
+                + "clock=SystemClock{}, "
+                + "resource=Resource{schemaUrl=null, attributes={service.name=\"otel-test\"}}, "
+                + "metricReaders=[PeriodicMetricReader{exporter=MockMetricExporter{}, intervalNanos=60000000000}], "
+                + "views=[RegisteredView{instrumentSelector=InstrumentSelector{instrumentType=null, instrumentName=instrument, meterName=null, meterVersion=null, meterSchemaUrl=null}, view=View{name=new-instrument, description=null, aggregation=DefaultAggregation, attributesProcessor=NoopAttributesProcessor{}}}]"
```

Comment:
It might make sense to not output `null` values for `RegisteredView` since it's so common